dev-cpp/gtest-parallel/gtest-parallel-20220621.ebuild

```
@@ -0,0 +1,34 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{13..14} )
+inherit python-r1
+
+DESCRIPTION="Run Google Test suites in parallel."
+HOMEPAGE="https://github.com/google/gtest-parallel"
+COMMIT="f4d65b555894b301699c7c3c52906f72ea052e83"
+SRC_URI="
+	https://github.com/google/gtest-parallel/archive/${COMMIT}.tar.gz
+	-> ${PN}-${COMMIT}.tar.gz
```

Just style but ebuilds typically do an extra tab for `->`  similarly to when they break up script lines to show the relation.

Also, when doing snapshots we often do `-> ${P}.tar.gz` albeit commit is fine too (keep what you prefer).
```suggestion
		-> ${P}.tar.gz
```